src/pages/search.module.scss

```diff
@@ -0,0 +1,5 @@
+.search-results {
+    h2 {
+        margin-bottom: 48px;
```

Comment:
You should use `rem` units I guess, (48 / 16) = 3rem (16px is the base font size)

Reply:
Yes, you are exactly right!